react/week2/Task/src/Task.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+const Task = ({ task, index, completeTask, removeTask }) => {
+    return (
+        <div
+            className="task"
+            style={{ textDecoration: task.completed ? "line-through" : "" }}
```

Comment:
Try not to use inline styles and instead give an element a class :) something like

```
<div className=`${task.completed ? "line-through" : ""}`>
```
and add class to css.